src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -99,11 +109,17 @@ public static String getReferenceSetID(String url) {
      */
     @Override
     public ReferenceBases getReferenceBases(final PipelineOptions pipelineOptions, final SimpleInterval interval) {
-        Utils.nonNull(pipelineOptions);
```

Comment:
Also, perhaps you want a one-arg overload of this method `getReferenceBases(interval)` that calls into `getReferenceBases(null, interval)` to hide the unused parameter from the client.